.github/workflows/build_and_test.yml

```diff
@@ -75,6 +75,7 @@ jobs:
         CC: ${{ matrix.cc }}
         LIBEXT: ${{ matrix.libext }}
       run: |
+        export PKG_CONFIG_PATH=/usr/local/opt/openssl@1.1/lib/pkgconfig:PKG_CONFIG_PATH
```

Comment:
PKG_CONFIG_PATH is set for MacOS platforms on line 42. Either remove this line if it's not needed in all platforms or remove line 42

Reply:
Removed the pre-existing line